### PR TITLE
fix(#90): Inject environment variables as test variables before scenario

### LINKED ISSUE #90: Add support for environment variable in feature file
I would like to have support for the environment variables in feature file:

1) propagate YAKS_NAMESPACE var to the running pod (we can reuse https://github.com/citrusframework/yaks/blob/master/pkg/cmd/test.go#L101)
Example:
```
Scenario:
  Given URL: http://user-report-system.${YAKS_NAMESPACE}.svc.cluster.local
  .....
```

2) support custom properties:
```
Scenario:
  Given URL: http://user-report-system.${CLUSTER_WILDCARD_DOMAIN}
  .....
```
Yaks command:
```
yaks test mytest.feature -e CLUSTER_WILDCARD_DOMAIN=apps.myocp.com
```

### PR DESCRIPTION
**Pull request by christophd**
Fixes #90

### CODE REVIEW

File: java/steps/yaks-standard/src/main/java/org/citrusframework/yaks/hooks/InjectEnvVarsHook.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    @Before
+    public void injectEnvVars(Scenario scenario) {
+        runner.run(new AbstractTestAction() {
+            @Override
+            public void doExecute(TestContext context) {
+                context.setVariable("SCENARIO_NAME", scenario.getName());
+                context.setVariable("SCENARIO_ID", scenario.getId());
+
+                injectEnvVar(context, "CLUSTER_WILDCARD_DOMAIN", "ns.svc.cluster.local");
```
Comment (bouskaJ) [inline java/steps/yaks-standard/src/main/java/org/citrusframework/yaks/hooks/InjectEnvVarsHook.java:44]: `ns.svc.cluster.local` was just an example and the `ns` is a shortcut for `namespace`. It should be `${YAKS_NAMESPACE}.svc.cluster.local`

Comment (christophd) [reply java/steps/yaks-standard/src/main/java/org/citrusframework/yaks/hooks/InjectEnvVarsHook.java:44]: haha I thought that would be a no-brainer so I started to switch of mine. Thanks for pointing to it - I will fix that

Comment (christophd) [reply java/steps/yaks-standard/src/main/java/org/citrusframework/yaks/hooks/InjectEnvVarsHook.java]: @bouskaJ solved

**Comment by christophd** [comment]
You can pass any env via CLI option

```
yaks test some-test --env YAKS_NAMESPACE=foo
```

These get set on the test CRD. I think we can auto add the env setting with the current namespace the test is working on.

**Comment by christophd** [comment]
I have added the YAKS_NAMESPACE variable by default so it gets injected to all test runtimes

**Pull request merged**